script/Ciel/Day 7/07_01_CIEL07_2_2.txt

```diff
@@ -831,19 +883,21 @@ Noel-sensei and the others still have to go around the city rooting out any rema
 [sha:d02ab94c2f6c80849d27725a422588bbe5c92fac]{
 -- Page 339, Offset 15632.
 -- 「それはどのくらいかかるものなんですか？」
-"And how long is that going to take?"
+"About how long does something like that usually take?"
 }
 [sha:029eea184a460b0e1e925aac85f296ced069ba4f]{
 -- Page 339, Offset 15633.
 -- 「んー、あの地下墓地の規模から見るに、下手すると半年くらいかかるかも。
-"Hmm, given the scale of those catacombs, it might take up to half a year in the worst case scenario.
+"Hmm, based on how extensive those catacombs were,
+ I'd say it could take up to half a year if things don't go our way.
 }
 [sha:d801fc86956d576f4b5c2c6fccfeed337dccc17f]{
 -- Page 339, Offset 15634.
 -- シエルさんが頑張らなければ、だけど。あの子、ワーカーホリックだからさ～。明日する仕事も今日やっちゃうタイプなのよね～」
-// there might be a more natural English expression for "to do tomorrow's work today", but I can't think of it.
-Well, as long as Ciel-san doesn't go hard.
- She's a real workaholic, you see. She's the type of person to do tomorrow's work today, and all that."
+// Expression here fairly close to Benny Franks quote, so subbed
+Well, that's unless Ciel-san really goes to town.
+ She's a total workaholic, you see.
+ Never leave til tomorrow what you can do today, and all that."
```

Comment:
or the old Ben Tsukihimate quote: "never do today what you can leave til tomorrow"